stl/inc/bit

```diff
@@ -127,13 +128,17 @@ extern int __isa_available;
 
 template <class _Ty>
 _NODISCARD int _Checked_x86_x64_countl_zero(const _Ty _Val) noexcept {
-    const bool _Have_lzcnt = __isa_available >= __ISA_AVAILABLE_AVX2;
     constexpr int _Digits  = numeric_limits<_Ty>::digits;
+
+#ifndef __AVX2__
+    const bool _Have_lzcnt = __isa_available >= __ISA_AVAILABLE_AVX2;
     // lzcnt (when it doesn't fall back to bsr) is defined correctly for zero
     // bsr has undefined output for zero
     if (!_Have_lzcnt && _Val == 0) {
         return _Digits;
     }
+#endif // !defined(__AVX2__

```

Comment:
Comment doesn't match preprocessor conditional. Normally we'd comment `// __AVX2__` whether the conditional is `#if __AVX2__` or `#ifdef __AVX2__` or `#ifndef __AVX2__` - the point is that they match so searching for the condition find the corresponding `#endif`.

Reply:
...and I've once again "requested changes" for something I could simply fix:
```suggestion
#endif // __AVX2__
```